### PR TITLE
Unifies v2 docs landing pages

### PR DESCRIPTION
**Pull request by michelle-luna**

### CODE REVIEW

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ order: 1
 * Contents
 {:toc}
 
+## What are the differences between CircleCI’s hosting options?
+
+- **Cloud** - CircleCI manages the setup, infrastructure, security and maintenance of your services. You get instant access to new feature releases and automatic upgrades, alleviating the need for manual work on an internal system.
+
+- **Server** - You install and manage CircleCI, through a service like AWS, behind a firewall that your team sets up and maintains according to your datacenter policy. You have full administrative control for complete customization and manage  upgrades as new versions are released.
```
Comment (ndintenfass) [inline jekyll/_cci2/faq.md]: This seems to now be outdated since we're not using Cloud and Server and proper nouns

File: jekyll/_cci2/index.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ permalink: /2.0/
 
 Welcome to CircleCI 2.0! The release of CircleCI 2.0 includes many improvements for faster performance and greater control. If you are new to CircleCI, check out the [Overview]({{ site.baseurl }}/2.0/about-circleci/) for how it works and then use the [Hello World]({{ site.baseurl }}/2.0/hello-world/) doc to start your first project build. Refer to the [Sample 2.0 config.yml File]({{ site.baseurl }}/2.0/sample-config/) to try out multiple jobs with workflow orchestration, then head over to the [2.0 Project Tutorial]({{ site.baseurl }}/2.0/project-walkthrough/).
 
+If you are an Administrator, see the [2.0 Administrator’s Overview]({{ site.baseurl }}/enterprise/overview/) document to get started and refer to the [Installing CircleCI 2.0 on Amazon Web Services with Terraform]({{ site.baseurl }}/enterprise/aws/) document for installation instructions.
```
Comment (ndintenfass) [inline jekyll/_cci2/index.md]: Should these still be links to the legacy Enterprise docs?

Comment (michelle-luna) [reply jekyll/_cci2/index.md]: no, I will do another sweep for links after I get the new files up

File: jekyll/_cci2/faq.md (hunk 1 above)
Comment (michelle-luna) [reply jekyll/_cci2/faq.md]: this is what Cheryl asked me to document, so I will keep it for now and fix tomorrow if needed

**Pull request merged**